react-native-masked-view/masked-view
# feat(example): add multiple component examples app

# Overview

Add an example app showing different ways to use the `MaskedView` component in Android and iOS.
The example app project has been created with [react-native-test-app](https://github.com/microsoft/react-native-test-app).

# Test Plan

1. Enter the `example` folder
2. Run `yarn` to install the dependencies
3. If you want to run the project on iOS, run `pod install --project-directory=ios` and then `yarn ios`
4. If you want to run the project on Android, just run `yarn android`
5. You should be able to select among three different examples
